### PR TITLE
create_data_ST

### PR DESCRIPTION
**Pull request by egolol**
changed the tex, pdf tutorials and test functions

### CODE REVIEW

File: tutorials/general/tut_gr_st/tut_gr_st.tex [hunk 1]
```diff
@@ -35,8 +35,8 @@ \section{Generation of Example Data}
 		This code can be used in the MatLab command line to generate the \fn{Example data ST XLS} folder to the \fn{structural} pipeline folder.
 	}
 ]
-test_ImporterGroupSubjectST_XLS ¥\circled{1}\circlednote{1}{generates the example structural XLS data folder.}¥
-test_ImporterGroupSubjectST_TXT ¥\circled{2}\circlednote{2}{generates the example structural TXT data folder.}¥
+create_data_SubjectST_XLS ¥\circled{1}\circlednote{1}{generates the example structural XLS data folder.}¥
```
Comment (giovannivolpe) [inline tutorials/general/tut_gr_st/tut_gr_st.tex]: change the name to 
create_data_ST_XLS()
also with parenthesis to signify these are function

File: braph2genesis/pipelines/structural/_ImporterGroupSubjectST_TXT.gen.m [hunk 2]
```diff
@@ -207,164 +207,7 @@ Subject ID (column 1), covariates (subsequent columns).
...
-    newTargets(s(t==source)) = 0;
-    newTargets(t(source, ~switchEdge)) = 0;
-
-    [~, ind] = sort(newTargets, 'descend');
-    t(source, switchEdge) = ind(1:nnz(switchEdge));
-end
-
-h = graph(s,t);
-end
+create_data_SubjectST_TXT()
```
Comment (giovannivolpe) [inline braph2genesis/pipelines/structural/_ImporterGroupSubjectST_TXT.gen.m]: there should be an if to check that this has not been created yet

File: braph2genesis/pipelines/structural/create_data_SubjectST_TXT.m [hunk 3]
```diff
@@ -0,0 +1,162 @@
+function create_data_SubjectST_TXT(data_dir)
```
Comment (giovannivolpe) [inline braph2genesis/pipelines/structural/create_data_SubjectST_TXT.m]: let's add also the random seed as a variable.

Also, add the comments

**Review by giovannivolpe — CHANGES_REQUESTED** [review]

File: braph2genesis/pipelines/structural/_ImporterGroupSubjectST_TXT.gen.m (hunk 2 above)
Comment (egolol) [reply braph2genesis/pipelines/structural/_ImporterGroupSubjectST_TXT.gen.m]: the if is inside the function no?

**Pull request merged**